### DISCUSSION TITLE
Converting safetensors to gguf

### DISCUSSION THREAD

**Discussion by Anish-Mahambare** [ggml-org/llama.cpp#8757] [Q&A]
![Screenshot from 2024-07-29 13-36-58](https://github.com/user-attachments/assets/2a76625e-40c8-4506-8815-44ff793e67b0)
![Screenshot from 2024-07-29 13-36-25](https://github.com/user-attachments/assets/e50c9fa2-8e54-44af-8b80-82e32a258f43)
Got above error when trying to convert safetensors to gguf, is there any fix?

**Comment by misutoneko** [comment]
Hi,

You've probably solved this by now, but the error message suggests that the architectures section is missing in your config.json.
Looks like you can grep Model.register to see a list of supported ones.
